lib/src/useStep/useStep.ts

```diff
@@ -0,0 +1,62 @@
+import { useCallback, useMemo, useState } from 'react'
+
+interface ReturnType {
+  actualStep: number
+  goToNextStep: () => void
+  goToPrevStep: () => void
+  reset: () => void
+  canGoToNextStep: boolean
+  canGoToPrevStep: boolean
+  setStep: (step: number) => void
+}
+
+function useStep(maxStep: number): ReturnType {
+  const [actualStep, setActualStep] = useState(1)
```

Comment:
Not very important, but I think that "current" is better than "actual".

```suggestion
const [currentStep, setCurrentStep] = useState(1)
```